### ISSUE TITLE
 training loss doesn't decrease and bad prediction

### ISSUE THREAD

**Issue by hakS07** [rishizek/tensorflow-deeplab-v3#33]
I tried to train DeeplabV3 pre-trained model on a new class(eyes iris)
my dtat=RGB iris image+gray-scale iris label:1500 images(300*400)
num_iteration=7000
crop_size=513
but the loss is unstable between 0.1 to 0.3
this example for a prediction
![Capture du 2019-05-27 22-56-27](https://user-images.githubusercontent.com/34586193/58440640-5a1fd080-80d3-11e9-8bba-a8e3988e83e6.png)

**Comment by TillBeemelmanns** [comment]
A Mask-RCNN is usually better for small object detection

**Comment by hakS07** [comment]
@TillBeemelmanns thank you
